src/api/app/views/webui2/webui/request/_request_history.html.haml

```diff
@@ -0,0 +1,47 @@
+%ul.list-unstyled
+  %li.media
+    - user = User.find_by_login(bs_request.creator)
+    = image_tag_for(user, size: 32, custom_class: 'mr-3', width: nil)
+    .media-body
+      %h5.mt-0.mb-1 #{bs_request.creator} (#{fuzzy_time(bs_request.created_at)})
+      created request
+      .media.mt-2
+        .media-body
+          - unless bs_request.description.blank?
+            .req-description-box
+              - if bs_request.description.length > 100
+                = simple_format(bs_request.description, class: 'collapsed-history', id: 'req-description')
+                = link_to('', '###', id: "show-full-req-description", class: 'collapsed-link', title: 'show more')
+              - else
+                = simple_format(bs_request.description)
+
+  - history.each_with_index do |element, index|
```

Comment:
you need to include elements and users to avoid extra queries :bulb:

Reply:
I think this is done in the request_controller through this line ```@history = @bs_request.history_elements.includes(:user)``` or do you mean something else?

Reply:
I meant comments and users. I think comments are not loaded and used... :thinking: